src/components/PoolStats/index.tsx

```diff
@@ -0,0 +1,146 @@
+import React, { useEffect } from "react";
+import { useDispatch } from "react-redux";
+import { Avatar } from "components/Avatar";
+import { Card } from "components/Card";
+import { StatItem } from "components/StatItem";
+import { fetchPoolStatsAction } from "ducks/poolStats";
+import { formatAmount } from "helpers/formatAmount";
+import { formatConversion } from "helpers/formatConversion";
+import { useRedux } from "hooks/useRedux";
+import {
+  LiquidityPoolReserve,
+  LiquidityPoolAssetInterval,
+} from "types/types.d";
+import "./styles.scss";
+
+export const PoolStats = () => {
+  const { poolAvatars, poolInfo, poolStats } = useRedux(
+    "poolAvatars",
+    "poolInfo",
+    "poolStats",
+  );
+
+  const dispatch = useDispatch();
+  const poolId = poolInfo.data?.id;
+
+  useEffect(() => {
+    if (poolId) {
+      dispatch(fetchPoolStatsAction(poolId));
+    }
+  }, [dispatch, poolId]);
+
+  const getAssetByAssetCode = ({
+    assets,
+    assetCode,
+  }: {
+    assets?: any[];
```

Comment:
Leaving `any` type here for now. It was a bit messy with the types because of the different formats of data. I'm hoping we can clean this up once we have formatted real data.